#-------------------------------------------------
# (patrol_times)
#-------------------------------------------------
patrol_times:
  ros__parameters:
    # Add as many poses in the patrol as desired (starts in pose_1)
    pose_1: [-1.0,-1.5,0.0,0.0,0.0,0.0,1.0]
    pose_2: [1.0,1.0,0.0,0.0,0.0,0.0,1.0]
